index.html

```diff
@@ -1,19 +1,94 @@
 <!DOCTYPE html>
 <html lang="en">
-<head>
-    <meta charset="UTF-8">
-    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
+  <head>
+    <meta charset="UTF-8" />
+    <meta
+      name="viewport"
+      content="width=device-width, initial-scale=1, shrink-to-fit=no"
+    />
     <title>Karma</title>
-    <link href='https://fonts.googleapis.com/css?family=Roboto:400,500,300' rel='stylesheet' type='text/css'>
-    <link rel="stylesheet" href="css/normalize.css">
-    <link rel="stylesheet" href="css/style.css">
-    <link rel="shortcut icon" type="image/x-icon"  href="favicon.ico">
-</head>
-<body>
+    <link
+      href="https://fonts.googleapis.com/css?family=Roboto:400,500,300"
+      rel="stylesheet"
+      type="text/css"
+    />
+    <link rel="stylesheet" href="css/normalize.css" />
+    <link rel="stylesheet" href="css/style.css" />
+    <link rel="shortcut icon" type="image/x-icon" href="favicon.ico" />
+  </head>
+  <body>
+    <header class="header">
+      <img
+        src="/img/karma-logo.svg"
+        id="logo-image-id"
+        class="logo-image"
+        alt="image_of_logo"
+      />
+      <nav class="navigation">
+        <a href="#" class="nav-links">Meet Karma</a>&nbsp;
+        <a href="#" class="nav-links">How it Works</a>&nbsp;
+        <a href="/store.html" class="nav-links">Store</a>&nbsp;
+        <a href="#" class="nav-links">Blog</a>&nbsp;
+        <a href="#" class="nav-links">Help</a>&nbsp;
+        <a href="#" class="nav-links">Logic</a>
+      </nav>
+    </header>
 
-    <!-- Add your HTML markup here -->
-    <!-- Remember: Use semantic HTML tags like <header>, <main>, <nav>, <footer>, <section> etc -->
-    <!-- All the images you need are in the 'img' folder -->
+    <main class="main-section">
+      <section class="backgroundImage-and-title">
+        <div class="intro">
+        <h1 class="title">Introducing Karma</h1>
+        <p class="intro-paragraph">Bring WiFi with you, everywhere you go</p>
+        <button class="learn-more-btn btn">Learn more</button>
+        </div>
+      </section>
 
-</body>
+      <section class="product-section">
```

Comment:
height of this section needs to be larger

Reply:
In mobile view it look perfect, but you in desktop view there issue in the height